manage-data/migrate/migrate-internal-indices.md

```diff
@@ -5,111 +5,35 @@ mapped_pages:
 applies_to:
   stack: ga
   deployment:
-    eck: unavailable
     ess: ga
-    ece: unavailable
-  serverless: unavailable
+    ece: ga
 products:
   - id: cloud-hosted
 ---
 
-# Migrate internal indices
+# Migrate system indices
 
-When you migrate your {{es}} data into a new infrastructure you may also want to migrate your {{es}} internal indices, specifically the `.kibana` index and the `.security` index.
+When you migrate your {{es}} data into a new infrastructure, you might also want to migrate system-level indices and data streams, such as those used by {{kib}} or security features (for example, `.kibana` and `.security`).
 
-There are two ways to migrate the internal {{es}} indices:
+Starting in {{es}} 8.0, you can use [feature states](/deploy-manage/tools/snapshot-and-restore.md#feature-state) to back up and restore all system indices and system data streams. This is the only available method for migrating this data.
```

Comment:
I'd still possibly add a little info about how this can be paired with any method for restoring the rest of your data